docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.5"
```

Comment:
what are we containerizing with this docker-compose?

Reply:
Deleting